settings/settings-module.php

```diff
@@ -29,9 +29,15 @@ class PLL_Settings_Module {
 	public $links_model;
 
 	/**
-	 * Stores if the module is active.
+	 * Key to use to manage the module activation state.
+	 * Possible values:
+	 * - An option key for a module that can be activated/deactivated.
+	 * - 'none' for a module that is always active.
```

Comment:
I know that I wrote that some days ago but it seems that's incorrect. Instead it looks like 'none' corresponds to "no settings available". Whether it is active or not is programmatically decided. See `PLL_Settings_Browser` for ex.

Reply:
You're right, I forgot this case. Done in [01e6886](https://github.com/polylang/polylang/pull/1420/commits/01e6886688f096099ab78716d813791289a8daf1).